{
	"schema_version": "1.6.0",
	"id": "USN-4245-1",
	"modified": "2020-01-21T16:41:42.168291Z",
	"published": "2020-01-21T16:41:42.168291Z",
	"related": [
		"CVE-2020-5390",
		"UBUNTU-CVE-2020-5390"
	],
	"summary": "python-pysaml2 vulnerability",
	"details": "It was discovered that PySAML2 incorrectly handled certain SAML files.\nAn attacker could possibly use this issue to bypass signature verification\nwith arbitrary data.\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:16.04:LTS",
				"name": "python-pysaml2",
				"purl": "pkg:deb/ubuntu/python-pysaml2@3.0.0-3ubuntu1.16.04.4?arch=source&distro=xenial"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.0.0-3ubuntu1.16.04.4"
						}
					]
				}
			],
			"versions": [
				"2.4.0-0ubuntu2",
				"3.0.0-3ubuntu1",
				"3.0.0-3ubuntu1.16.04.1",
				"3.0.0-3ubuntu1.16.04.3"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-pysaml2",
						"binary_version": "3.0.0-3ubuntu1.16.04.4"
					},
					{
						"binary_name": "python-pysaml2-doc",
						"binary_version": "3.0.0-3ubuntu1.16.04.4"
					},
					{
						"binary_name": "python3-pysaml2",
						"binary_version": "3.0.0-3ubuntu1.16.04.4"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-4245-1.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:18.04:LTS",
				"name": "python-pysaml2",
				"purl": "pkg:deb/ubuntu/python-pysaml2@4.0.2-0ubuntu3.1?arch=source&distro=bionic"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "4.0.2-0ubuntu3.1"
						}
					]
				}
			],
			"versions": [
				"3.0.0-3ubuntu2",
				"4.0.2-0ubuntu2",
				"4.0.2-0ubuntu3"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-pysaml2",
						"binary_version": "4.0.2-0ubuntu3.1"
					},
					{
						"binary_name": "python-pysaml2-doc",
						"binary_version": "4.0.2-0ubuntu3.1"
					},
					{
						"binary_name": "python3-pysaml2",
						"binary_version": "4.0.2-0ubuntu3.1"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-4245-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://ubuntu.com/security/notices/USN-4245-1"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2020-5390"
		}
	]
}
